src/libs/reportUtils.js

```diff
@@ -1,5 +1,6 @@
 import _ from 'underscore';
 import Str from 'expensify-common/lib/str';
+import lodashGet from 'lodash/get';
```

Comment:
`underscore` has the same functionality and syntax if you update the package

```js
_.get(item, path, defaultValue);
```

Reply:
Interesting! We should consider it, but not necessary for now since we have a lot of the lodash usages already.